src/components/navbar/NavSection.jsx

```diff
@@ -269,7 +270,7 @@ const NavSection = ({
           </div>
         )}
       </Droppable>
-      <div className="d-flex justify-content-between mt-4">
+      <Box d-flex justify-content-between mt="2rem">
```

Comment:
Don't you still need the `className` here?

Reply:
This is for the sidebar which doesn't need to follow the existing template styles - so i think we can drop the `className` as long as the styling is fine

Reply:
ok then pls drop the `d-flex...` as those are unused and leaving them in is confusing

Reply:
Removed in https://github.com/isomerpages/isomercms-frontend/pull/1466/commits/eb6e949f13de868115b0ad726338e1d6209601c9